README.md

```diff
@@ -46,3 +47,19 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## Response:
+
+The worst-case for the time complexity of this implementation suggests that new values will be memoized but similar instances of the function will never reoccur and those memoized values will never have the opportunity to be used so we will have to do the whole function for every order of nodes. The function for all possible combinations without repetition would be $\Theta(|V|!)$ where $|V|$ is the number of vertices.
```

Comment:
Are you indexing into your cache using permutations of the cities?

Reply:
Yes, with start. Am I missing something?

Reply:
Really? `nodesLeft` is a permutation of all cities?